sites/main-site/src/content/blog/2025/springcleaning.mdx

```diff
@@ -0,0 +1,122 @@
+
+Prior the spring cleaning, we split up into different groups that were focused on particular tasks:
+
+- nf-core/pipelines
+- nf-core/modules
+- nf-core/documentation
+- nf-core/configs
+- Hackathon preparation
+- Core Team
+
+The teams were extremely productive throughout the week, as can be seen just with merged PRs alone from this [pullpo.io](https://pullpo.io/) graph:
+
+![Bar chart of number of PRs merged by the maintainers team between March 10th-16th, with a total of 77 merged PRs throughout the week](src/assets/images/blog/springcleaning2025/maintainers-team-merged-stats.png)
+
+## Progress
+
+### Pipelines
+
+📑 The pipelines cleaning team were tasked with checking the state of each pipeline's repository.
+This involved assessing the number of stale issues, branches, PRs etc, as well as checking minor things such as that tags and descriptions were all present and correct.
+The pipelines team were also evaluating the development status of the pipeline. If a pipeline seemed to be very 'quiet', they began to contact the lead developers regarding whether the pipeline is still under active development or maintenance.
+
+✅ By the end of the spring cleaning, almost half of the nf-core pipelines were assessed.
+Most of them were evaluated to be in good shape.
+Fun fact: of the 50 assessed, 5 were still written in Nextflow DSL1!
+Around 10 of them appear to be no longer maintained, and this list was passed to the core team for further evaluation about their status and whether they should be possibly archived.
+
+🛠️ A yearly check seemed to be enough even if not all pipelines were checked this time round.
+For next year's spring cleaning, Jonas and Florian requested we ensure the team has more maintainers to work on this task.
+
+### Modules
+
+📑 The modules team were tasked with tackling the 🦣 modules repository of more than 1,400 modules, and hundreds of open issues and PRs.
+
+✅ An **astounding** number of issues and PRs were reviewed and closed.
+Throughout the week this team reviewed 162 PRs, closing at least 62 of them, they also reviewed 213 issues, closing 72.
+Special shout-out to the MVPs of Famke (52 PRs and 108 issues!), Luisa, and Simon who started _early_ and powered through the huge number of issues and PRs.
+
+🛠️ During our wrap up session we discussed a number of options to better handle the modules repository using automation.
+For example, we discussed potentially making better use of GitHub tags and [types](https://docs.github.com/en/issues/tracking-your-work-with-issues/configuring-issues/managing-issue-types-in-an-organization), to help both prepare spring-cleaning task boards, but also to potentially 'auto close' stale PRs and issues after a certain amount of time.
+We also discussed maybe requiring all new modules proposals/issues to be made on pipeline repositories and linked to the modules repository, to ensure the module has a direct application, and thus the repository issues does not get filled up with just 'ideas'.
```

Comment:
Just a comment on the discussed point here: nf-core modules go now beyond nf-core pipelines, so this would hinder the spread of our standards and lead to more split up efforts.

Reply:
We can discuss in the next maintainers meeting :)